src/app/components/mnoe-tasks/mnoe-tasks.coffee

```diff
@@ -209,7 +209,7 @@
 
     # Formats dates yesterday & beyond differently from today
     expandingDateFormat = (value)->
-      dateFormat = if moment(value) < moment().startOf('day') then 'MMMM d' else 'h:mma'
+      dateFormat = if moment.utc(value) < moment().startOf('day') then 'MMMM d' else 'h:mma'
```

Comment:
This is just building the date format string for a $filter('date'), it is pretty simple imo, if the value (timestamp) is less than today (aka yesterday), format the date differently.